server/routes/plan-summary/locale.json

```diff
@@ -13,6 +13,9 @@
       "changedGoal": "You changed a goal in {{ subject.possessiveName }} plan.",
       "removedGoal": "You removed a goal from {{ subject.possessiveName }} plan."
     },
+    "agreePlan": "{{ subject.givenName }} agreed to their plan on {{ plan.agreementDate }}",
+    "notAgreePlan": "Plan created on {{ plan.creationDate }}",
+    "lastUpdatedPlan": "Last updated on {{ plan.updatedDate }} by ---",
```

Comment:
Does the --- here need replacing with a variable? is it the practitioner's name that needs putting here?

Reply:
Yes I think this will require backend update. So just left as placeholder but ATM this message will never be displayed.

Reply:
Ah yeah, great point - we can't use the practitioner details from the handover, it'll have to be something passed when we update the plan/goals and then stored. I'll approve for now and we can address this later :)